[DOC-2781] Change ChaosHub to chaos hub (#994)
harness/developer-hub: release-notes/chaos-engineering.md

* Chaos configuration step in **Pipelines** shows the name of the chaos experiment (CHAOS-986)
    * This release shows the name of the experiment instead of showing the experiment Id in the chaos configuration setup step in **Pipelines**. This helps you identify experiments with ease.

* Search functionality when selecting experiments from chaos hub (CHAOS-1050)

>     * This release adds search functionality when selecting an experiment template from chaos hub. You can also filter the experiments you wish to view or select from the chaos hub. This allows you to select and run your experiment without having to search through multiple experiments.

Reviewer: ```suggestion
    * This release adds search functionality when selecting an experiment template from chaos hub. You can also filter the experiments you want to view or select from the chaos hub. This allows you to select and run your experiment without searching multiple experiments.
```

    * This release moves the **Set fault weights** tab, which was previously a separate tab, into the **Tune fault** tab. This allows you to tune the fault parameters and set fault weights in a single step rather than navigating through multiple tabs.

* Support for the GitLab connector (CHAOS-35)

>     * This release introduces a new connector, namely the GitLab connector, when you wish to connect to a chaos hub.

Reviewer: ```suggestion
    * This release introduces a new connector called GitLab connector to connect to a chaos hub.
```

 
* In **Pipeline**, when you tried to select an experiment in the chaos experiments page, the pagination section overflowed. This issue has been fixed so that the chaos experiments plage shows two buttons: **Prev** and **Next** to navigate through the pages. (CHAOS-1045)  
 

> * In chaos hubs, the number of experiments in the category tab for the chaos experiments overflowed. This issue has been fixed. (CHAOS-1053)

Reviewer: This issue is fixed.

* Support for deployment on existing chaos infrastructure (CHAOS-954)
    * This release adds support for deploying your chaos infrastructures on clusters that use the existing (deployed) Harness delegates (also known as the brownfield method of deployment). You can select the connector that points to the required delegate and other details like installation mode, service account name, and namespace, after which the YAML manifest is generated and sent over to the cluster instead of being downloaded on your system. Once the delegate receives the manifest, it deploys your chaos infrastructure on the selected cluster. Currently, you can deploy the chaos infrastructure by using only account-level delegates.

* Details of an experiment are prefilled when adding it to a chaos hub (CHAOS-989)

>     * Instead of forcing you to re-enter details, this release prefills the details of the experiment that you want to add to a chaos hub. You can simply navigate to the experiment and click **Add to ChaosHub**. The resulting screen displays the name of the experiment, a description (optional), and tags (optional). You can add your experiment to the chaos hub of your choice by clicking **Save**.

Reviewer: ```suggestion
    * Instead of forcing you to re-enter details, this release prefills the details of the experiment that you want to add to a chaos hub. You can simply navigate to the experiment and select **Add to ChaosHub**. The resulting screen displays the name of the experiment, a description (optional), and tags (optional). You can add your experiment to the chaos hub of your choice by selecting **Save**.
```

* One sync retry to connect to a disconnected chaos hub (CHAOS-999)

>     * A chaos hub that is disconnected does not list any faults or experiments. This release adds a feature such that when you click on a disconnected chaos hub, HCE tries to synchronize and connect to the chaos hub at least once. 

Reviewer: ```suggestion
    * A chaos hub that is disconnected does not list any faults or experiments. In this release, when you click a disconnected chaos hub, HCE tries to synchronize and connect to the chaos hub at least once. 
```

### Fixed issues
* Searching for chaos experiments by using the search bar showed only those experiments that had been run at least once. Now, when you search for an experiment, the search results include those experiments that were aborted and experiments that were saved but not run. (CHAOS-916)
* When specifying the target application parameters through a YAML manifest, if you left some parameters empty, the user interface of the target application page would stop responding. This has been fixed so that, irrespective of the values that you enter in the YAML manifest, you can change the values of the target application on the user interface. (CHAOS-970)

> * In a chaos experiment, the fault library would incorrectly show fault categories and fault labels even when the hub had been disconnected. The fault library would persist data from the previously selected chaos hub. This has now been fixed. A disconnected chaos hub now displays the message “No faults found in the selected hub.” (CHAOS-971)

Reviewer: ```suggestion
* In a chaos experiment, the fault library incorrectly showed fault categories and fault labels even when the hub was disconnected. The fault library persisted data from the previously selected chaos hub. This is now fixed. A disconnected chaos hub now displays the message “No faults found in the selected hub.” (CHAOS-971)
```

> * On the chaos hub screen, you could not scroll through the list of hubs from any location on the screen. This issue has been fixed by moving the scroll bar to the extreme right of the screen. (CHAOS-964)

Reviewer: ```suggestion
* On the chaos hub screen, you could not scroll through the list of hubs from any location on the screen. This issue is now fixed by moving the scroll bar to the extreme right of the screen. (CHAOS-964)
```

* If you hovered over a probe, its details would overflow if they were too long. Now, it has been fixed. (CHAOS-990)

> * Any increase in the number of chaos faults that you wished to view on a single page in a chaos hub would result in a blank page. Now, it has been fixed. (CHAOS-984)

Reviewer: Does this mean the following?
Adding new chaos faults on a single page in a chaos hub would result in a blank page. Now, this is fixed. (CHAOS-984)

> * When a chaos experiment was imported into the chaos hub, it would not be logged as an audit event and would not be displayed on the user interface. It has been fixed. (CHAOS-779)

Reviewer: ```suggestion
* When a chaos experiment was imported into the chaos hub, it wasn't logged as an audit event and didn't appear on the user interface. It is now fixed. (CHAOS-779)
```

### Fixed issues

> * Enterprise chaos hub appeared in the search results irrespective of the terms searched. Now it has been fixed.

Reviewer: It is fixed. I see a lot of has been in release notes. Keep it that way if its the norm. I would prefer using present tense.
